js/whitebit.js

```diff
@@ -615,11 +706,438 @@ module.exports = class whitebit extends Exchange {
+            const provider = this.safeNumber2 (params, 'provider');
+            if (provider === undefined) {
+                throw new ArgumentsRequired (this.id + ' fetchDepositAddress() requires a provider when the ticker is fiat');
+            }
+            request['provider'] = provider;
+            const amount = this.safeNumber2 (params, 'amount');
+            if (amount === undefined) {
+                throw new ArgumentsRequired (this.id + ' fetchDepositAddress() requires an amount when the ticker is fiat');
+            }
+            request['amount'] = amount;
+            const uniqueId = this.safeValue (params, 'uniqueId');
+            if (uniqueId === undefined) {
+                throw new ArgumentsRequired (this.id + ' fetchDepositAddress() requires an uniqueId when the ticker is fiat');
+            }
+        }
+        const response = await this[method] (this.extend (request, params));
+        // Response for fiat
+        // {
+        //     "url": "https://someaddress.com"                  // address for deposit
+        // }
+        // Response for crypo
+        // {
+        //     "account": {
+        //         "address": "GDTSOI56XNVAKJNJBLJGRNZIVOCIZJRBIDKTWSCYEYNFAZEMBLN75RMN",        // deposit address
+        //         "memo": "48565488244493"                                                      // memo if currency requires memo
+        //     },
+        //     "required": {
+        //         "fixedFee": "0",                                                              // fixed deposit fee
+        //         "flexFee": {                                                                  // flexible fee - is fee that use percent rate
+        //             "maxFee": "0",                                                            // maximum fixed fee that you will pay
+        //             "minFee": "0",                                                            // minimum fixed fee that you will pay
+        //             "percent": "0"                                                            // percent of deposit that you will pay
+        //         },
+        //         "maxAmount": "0",                                                             // max amount of deposit that can be accepted by exchange - if you deposit more than that number, it won't be accepted by exchange
+        //         "minAmount": "1"                                                              // min amount of deposit that can be accepted by exchange - if you will deposit less than that number, it won't be accepted by exchange
+        //     }
+        // }
+        const url = this.safeString (response, 'url');
+        const account = this.safeValue (response, 'account', {});
+        const address = this.safeString (account, 'address', url);
+        const tag = this.safeString (account, 'memo');
+        this.checkAddress (address);
+        this.checkAddress (address);
```

Comment:
One of the above two lines is here by accident )

Reply:
I wanted to be double sure 😄 

removed